app/jobs/vimeo/set_embed_links.rb

```diff
@@ -0,0 +1,12 @@
+module Vimeo
+  class SetEmbedLinks < ApplicationJob
+    queue_as :default
+
+    def perform(current_school, video_id)
+      current_school.domains.pluck(:fqdn).map do |fqdn|
```

Comment:
I think this arg should be named `school`, not `current_school`, since _current_ has no meaning in an asynchronous job. :-)

```suggestion
    def perform(school, video_id)
      school.domains.pluck(:fqdn).map do |fqdn|
```